website/docs/sys-mgmt/service-accounts.md

```diff
@@ -0,0 +1,108 @@
+---
+title: Service Accounts
+sidebar_label: Service Accounts
+---
+
+# Service Accounts
+
+Service accounts are specialized user accounts designed for machine-to-machine authentication and automation purposes rather than interactive human use. They're ideal for integrating authentik with external systems, APIs, and services.
+
+## Types of Service Accounts
+
+authentik distinguishes between two types of service accounts:
+
+1. **User-created Service Accounts**: Created by administrators for integrating with external systems or for automation purposes.
+2. **Internal Service Accounts**: Created and managed automatically by authentik for internal purposes, such as outpost communications. These cannot be created manually.
+
+## Creating a Service Account
+
+To create a service account:
+
+1. In the authentik **Admin interface**, navigate to **Directory** > **Users**.
+2. Click the **Create Service Account** button.
+3. Configure the following settings:
+    - **Username**: The user's primary identifier (150 characters or fewer).
+    - **Create Group**: Enabling this toggle will create a group named after the user, with the user as a member.
+    - **Expiring**: If selected, the token will expire and be automatically rotated upon expiration.
+    - **Expires on**: Sets the expiration date (defaults to 1 year from the creation date).
+4. Click **Create Service Account**.
+
+After creating the service account, you'll see a confirmation screen that shows the username and generated password (token). The password is a long, randomly generated string that will be valid for 360 days by default. Make sure to copy this information somewhere secure as you'll need it for authentication.
+
+## Authentication with Service Accounts
+
+Service accounts authenticate using [HTTP Basic Authentication](https://datatracker.ietf.org/doc/html/rfc7617). The username and password (token) generated during account creation are used as credentials.
+
+## Managing Service Account Tokens
+
+Tokens for service accounts can be managed through the authentik **Admin interface**:
+
+1. Navigate to **Admin Interface** > **Directory** > **Tokens and App passwords**.
+2. Here you can view, create, copy, delete, and manage tokens.
+
+### Creating New Tokens
+
+To create a new token for a service account:
+
+1. Click the **Create** button.
```

Comment:
We need to at least tell them what page to start from, maybe how to get to that page. Something like "On the **Tokens** page (**Customize** -> **Tokens and Apps passwords**), click **Create**.

Also, we don't say "button"... just "Click on... and then the label.